test/common/testlib.py

```diff
@@ -1178,13 +1180,13 @@ def assert_pixels(self, selector: str, key: str,
             for layout in self.layouts:
                 if layout["name"] not in skip_layouts:
                     self.set_layout(layout["name"])
-                    time.sleep(wait_delay)
```

Comment:
Did you deliberately take this out? This was for letting the browser settle down the page after a layout switch.

That said,perhaps that should move after the `_set_direction("rtl")` below. I still occasionally see test failures where the supposed RTL dialog is still LTR (sorry, I don't have an example URL at hand)

Reply:
Yes, the intention was to move the delay, so that it happens later, after layout switching, mocking, _and_ scrolling into view. 

Do you say it should always happen, even with `wait_animations == False`? Is anyone actually using `wait_animations = False`?

Reply:
It's fine, I just wanted to know if it was deliberate. Moving the waiting there should make it more robust indeed. I don't see any project using wait_animations=False, we could just as well drop that option.

Reply:
Removing the delay here did indeed seem to trigger some failures elsewhere, see the additional two commits. But I hope it's not a general thing; I hope we don't need to wait after layout changes (and then after scrolling again).